.github/workflows/release_build.yaml

```
@@ -34,12 +34,9 @@ jobs:
       - name: Patch version for beta-builds
         run: |
           echo -n $(cat version.txt | sed -e "s/.*/&/") >> ./version_patched
-          echo -n "-$BRANCH_NAME-" >> ./version_patched
           echo -n $(date '+%m.%d.%Y-%H.%M.%S') >> ./version_patched
```

Чёрт, я немного продолбался. Давай добавим сюда -, чтобы отделить этот префикс от номера версии?
```suggestion
          echo -n -$(date '+%m.%d.%Y-%H.%M.%S') >> ./version_patched
```